.orchestra/support/revng-distributable/install.sh

```diff
@@ -106,7 +106,7 @@ rm "$TMP_IDX"
 
 echo "Generating checksums"
 cd "$DESTDIR/$ORCHESTRA_ROOT/revng"
-find root -type f -exec sha256sum {} \; > checksums.sha256
+find root -type f -print0 | xargs -0 -P"$(nproc)" -n100 sha256sum > checksums.sha256
```

Comment:
You should use `parallelism` from orchestra configuration. Annoying since it's a standalone script... Maybe you can export that in a global variable.

Reply:
We're in luck, we have `$JOBS`